pkg/kt/azure.go

```diff
@@ -0,0 +1,60 @@
+package kt
+
+/**
+Helper functions to make working with Azure easier.
+*/
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets"
+)
+
+var (
+	azureClient *azsecrets.Client
+)
+
+const (
+	AzureKVPrefix  = "azure.kv."
+	AzureErrPrefix = "AzureError: "
+	azureTimeout   = 10 * time.Second
+)
+
+func loadViaAzureKeyVault(key string) string {
+	if azureClient == nil {
+		keyVaultName := os.Getenv("KT_AZURE_KEY_VAULT_NAME") // Must be set.
+		if keyVaultName == "" {
+			log.Printf("ENV Var 'KT_AZURE_KEY_VAULT_NAME' must be set")
+			return "ENV Var KT_AZURE_KEY_VAULT_NAME must be set"
+		}
+
+		keyVaultURL := fmt.Sprintf("https://%s.vault.azure.net/", keyVaultName) // Should this be hard coded?
```

Comment:
Looks like these can possibly have different private external endpoints (https://docs.microsoft.com/en-us/azure/key-vault/general/private-link-service?tabs=portal) but probably fine for now.

Reply:
Updated to allow a custom endpoint if you really want one.